python/ray/serve/tests/test_logging.py

```diff
@@ -72,6 +72,7 @@ def _get_expected_replica_log_content(replica_id: ReplicaID):
     return f"{app_name}_{deployment_name} {replica_id.unique_id}"
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="does not work on windows")
```

Comment:
If you disable our log rotation, this test should just work, no need to skip.

Reply:
ok